modules/ossm-rn-known-issues.adoc

```diff
@@ -30,6 +30,9 @@ These limitations exist in {SMProductName}:
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-1211[OSSM-1211]
+OpenShift Service Mesh Federation failover Transport Layer Security (TLS) error when setting `importAsLocal` to `true` in two separate federated meshes.
```

Comment:
I think we need to be clearer here, the problem is not so much that there is an error message as that Failover doesn't work.  Maybe something like, "Configuring Federated service meshes for failover does not work as expected.  There is no workaround at this time."